tests/std/tests/GH_002558_format_presetPadding/env.lst

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# clang-cl doesn't support /presetPadding this is why we don't use concepts_20_matrix.lst

```

Comment:
It's helpful to folks who need to flow changes through the entire tree of `env.lst`s to know the relationship between files:
```suggestion
# This is concepts_20_matrix.lst +/presetPadding with clang configs disabled (clang-cl doesn't support /presetPadding)
```
What's obvious now won't necessarily be so to future us.